dwave/system/utilities.py

```diff
@@ -89,3 +93,55 @@ def get(name, default=False):
     @classproperty
     def hss_solver_config_override(cls):
         return cls.get('hss_solver_config_override')
+
+
+def schedule_with_anneal_offset(
+        schedule_csv_filename: str,
```

Comment:
I think we can mirror the input types of [np.loadtxt](https://numpy.org/devdocs/reference/generated/numpy.loadtxt.html) here.

Reply:
[This commit ](https://github.com/dwavesystems/dwave-system/pull/568/commits/12c16eb3aef16cf1cb1fd44a9572b8ab6e1b6620) switches the code to accepting array-like inputs instead of a CSV file.